backend/danswer/chat/load_yamls.py

```diff
@@ -82,9 +82,9 @@ def load_personas_from_yaml(
                 persona_id=(-1 * p_id) if p_id is not None else None,
                 name=persona["name"],
                 description=persona["description"],
-                num_chunks=persona.get("num_chunks")
-                if persona.get("num_chunks") is not None
-                else default_chunks,
+                num_chunks=persona.get("num_chunks", default_chunks),
```

Comment:
if persona.get("num_chunks") is None, it will just be None, it won't use default_chunks
better to use the "or" I think